QSWebAPI.sln

```diff
@@ -0,0 +1,16 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Please separate the project from the solution, you can take one of my templates as an example. We don't just want one project in our solution. Now we are locking the solution to one project.